### PR TITLE
Revise specialists concept types

### PR DESCRIPTION
**Pull request by asparuhft**
# Description

## What

Revise specialists concept types

## Why

JIRA Ticket [UPPSF-5514](https://financialtimes.atlassian.net/browse/UPPSF-5514)

## Anything, in particular, you'd like to highlight to reviewers

Mention here sections of code which you would like reviewers to pay extra attention to .E.g

_Would appreciate a second pair of eyes on the test_  
_I am not quite sure how this bit works_  
_Is there a better library for doing x_  

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [x] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

[UPPSF-5514]: https://financialtimes.atlassian.net/browse/UPPSF-5514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by agrancharova — APPROVED** [review]

**Review by angelraynovft — DISMISSED** [review]

File: helm/concept-rw-elasticsearch/templates/deployment.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
         - name: ELASTICSEARCH_INDEX
           value: all-concepts
         - name: ELASTICSEARCH_WHITELISTED_CONCEPTS
-          value: "genres,topics,sections,subjects,locations,brands,organisations,people,alphaville-series,memberships,sv-categories,sv-provisions,fta-brands,fta-genres,fta-topics"
+          value: "genres,topics,sections,subjects,locations,brands,organisations,people,alphaville-series,memberships,fta-topics"
         - name: ELASTICSEARCH_ENDPOINT
```
Comment (agrancharova) [inline helm/concept-rw-elasticsearch/templates/deployment.yaml:45]: lets keep fta-brands and fta-genres in the configuration

**Review by agrancharova — CHANGES_REQUESTED** [review]
lets keep fta-brands and fta-genres in the configuration

**Review by agrancharova — DISMISSED** [review]

**Review by agrancharova — DISMISSED** [review]

**Review by agrancharova — APPROVED** [review]

**Review by ManoelMilchev — COMMENTED** [review]
This branch is 7 commits behind master, please rebase.

**Comment by asparuhft** [comment]
> This branch is 7 commits behind master, please rebase.

Done. Thanks

**Review by ManoelMilchev — APPROVED** [review]

**Comment by agrancharova** [comment]
Can you rebase, instead of merging, so we avoid the merge commit

**Comment by asparuhft** [comment]
> Can you rebase, instead of merging, so we avoid the merge commit

Done

**Pull request merged**